Makefile

```
@@ -1,19 +1,34 @@
 CC = cl
 LD = link
+RC = rc 
 
 EXEC = lightwm.exe
 DLL = lightwm_dll.dll
+RESOURCES=wm_dll_resources.obj
 
-all: clean $(DLL) $(EXEC)
+# Boiler plate
+CFLAGS  = /EHsc /Zi
+LDFLAGS = 
 
-$(EXEC): wm.c
-	$(CC) wm.c tiling.c error.c user32.lib /link /out:$(EXEC)
+EXE_SRCS = wm.c tiling.c error.c
+DLL_SRCS = error.c config.c keyboard.c wm_dll.c
 
-$(DLL): wm_dll.obj
-	$(LD) wm_dll.obj user32.lib /dll /out:$(DLL)
+EXE_OBJS = $(EXE_SRCS:.c=.obj)
+DLL_OBJS = $(DLL_SRCS:.c=.obj)
 
-wm_dll.obj: wm_dll.c
-	$(CC) /c wm_dll.c
+all: $(DLL) $(EXEC)
```

```suggestion
all: clean $(DLL) $(EXEC)
```
For now also cleaning (gonna leave it like this for a couple of weeks), since I changed the EXE name and I want to save confusion of people using the old EXE by mistake.